src/content/en/2021/performance.md

```diff
@@ -1,20 +1,534 @@
+## Introduction
+
+Performance is important for user experience. Slow-to-load and slow-to-respond websites frustrate users and cause lost conversions. This is the first year that the <a hreflang="en" href="https://web.dev/vitals/">Core Web Vitals</a> have contributed to Google search rankings. As such, we've seen greater interest in improving website performance which is great news for users.
+
+*What are our top takeaways from this year's report?* First, we still have a long way to go in providing a good user experience. For example, faster networks and devices have not yet reached the point where we can ignore how much JavaScript we deliver to a site; and, we may never get there. Second, sometimes we misuse new features for performance, resulting in poorer performance. Third, we need better metrics for measuring interactivity, and those are on the way. And fourth, CMS- and framework-level work on performance can significantly impact user experience for the top 10M websites.
+
+*What's new this year?* We're excited to share performance data by traffic ranking for the first time. We also have all the core performance metrics from previous years. Finally, we added a deeper dive into the Largest Contentful Paint (LCP) element.
+
+### Notes on Methodology
+
+One thing that makes the performance chapter different from the others is that we rely heavily on the <a hreflang="en" href="https://developers.google.com/web/tools/chrome-user-experience-report">Chrome User Experience Report</a> (CrUX) for our analyses. Why? If our number one priority is user experience, then the best way to measure performance is with real user data (real user metrics, or RUM for short).
+
+> The Chrome User Experience Report provides user experience metrics for how real-world Chrome users experience popular destinations on the web. -<a hreflang="en" href="https://developers.google.com/web/tools/chrome-user-experience-report">Chrome User Experience Report</a>
+
+CrUX data only provides high-level field/RUM metrics and only for the Chrome browser. Additionally, CrUX reports data by origin, or website, instead of by page.
+
+We supplement our CrUX RUM data with lab data from WebPageTest in HTTP Archive. WebPageTest includes very detailed information about each page, including the full Lighthouse report. Note that WebPageTest measures performance in <a hreflang="en" href="https://almanac.httparchive.org/en/2021/methodology#webpagetest">California and Oregon</a>. The performance data in CrUX is global since it represents real user page loads.
+
+When comparing performance year-over-year, keep in mind that:
+
+- The Cumulative Layout Shift (CLS) calculation has changed since 2020.
+- The First Contentful Paint (FCP) thresholds ("good", "needs improvement", and "poor") have changed since 2020.
+- Last year's report was based on August 2020 data, and this year's report was based on the July 2021 run.
+
+Read the full [methodology](./methodology) for the Web Almanac to learn more.
```

Comment:
I think removing the last item as a bullet makes it easy to skip over. So will make the other changes but put it back as a bullet.

Reply:
Was trying to separate CWV changes (and link for that), from Web Almanac changes (and link for that).

I'm also not convinced we even need to mention the July -> August thing. I think it adds unnecessary confusion to this chapter and is best handled in Methodology page. One month doesn't make a lot of difference and it was done for project reasons rather than any particular measurement reasons.